conans/client/manager.py

```diff
@@ -128,14 +128,17 @@ def install(self, reference, current_path, remote=None, options=None, settings=N
         @param remote: install only from that remote
         @param options: written in JSON, e.g. {"compiler": "Visual Studio 12", ...}
         """
+        reference_given = True
```

Comment:
reference = None was used to indicate no reference was given, but maybe this name indicates intention more clearly, yes.